substrate/frame/identity/src/lib.rs

```diff
@@ -227,6 +247,10 @@ pub mod pallet {
 		#[pallet::constant]
 		type MaxUsernameLength: Get<u32>;
 
+		#[cfg(feature = "runtime-benchmarks")]
+		/// A set of helper functions for benchmarking.
```

Comment:
The comment normally goes above the feature gate.

Can you also please include a line what the default config `()` means?

Reply:
That was copy-pasted from elsewhere in the repository, so for sure there will be more cases around in other parts not covered in this PR. Just the heads up.

Addressed in [7bb7da3](https://github.com/paritytech/polkadot-sdk/pull/8179/commits/7bb7da31acfcf6d1c9b189164884bcdf3d79cc6c).

Reply:
> Can you also please include a line what the default config () means?

Missed that part. Done in [926e317](https://github.com/paritytech/polkadot-sdk/pull/8179/commits/926e3176aba82f098a720b13cc603d3468d90802).